doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -661,57 +624,23 @@ the conversion in hypotheses :n:`{+ @ident}`.
             Goal 1 + 0 = 1.
             unfold Nat.add.
 
-   .. tacv:: unfold @qualid in @goal_occurrences
-
-      Replaces :n:`@qualid` in hypothesis (or hypotheses) designated
-      by :token:`goal_occurrences` with its definition and replaces
-      the hypothesis with its :math:`\beta`:math:`\iota` normal form.
-
-   .. tacv:: unfold {+, @qualid}
-
-      Replaces :n:`{+, @qualid}` with their definitions and replaces
-      the current goal with its :math:`\beta`:math:`\iota` normal
-      form.
-
-   .. tacv:: unfold {+, @qualid at @occurrences }
-
-      The list :token:`occurrences` specify the occurrences of
-      :n:`@qualid` to be unfolded. Occurrences are located from left
-      to right.
-
       .. exn:: Bad occurrence number of @qualid.
          :undocumented:
 
       .. exn:: @qualid does not occur.
          :undocumented:
 
-   .. tacv:: unfold @string
-
-      If :n:`@string` denotes the discriminating symbol of a notation
-      (e.g. "+") or an expression defining a notation (e.g. `"_ +
-      _"`), and this notation denotes an application whose head symbol
-      is an unfoldable constant, then the tactic unfolds it.
```

Comment:
This text seems to have disappeared, and the text that replaced it seems to be less specific about how to get a constant out of `@string`.

Reply:
See what I wrote. (FWIW I'm not a fan of "denotes".)